snmp/datadog_checks/snmp/data/default_profiles/cyberpower-pdu.yaml

```diff
@@ -0,0 +1,375 @@
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.6.3.4.1.14
+          name: ePDU2DeviceStatusPowerSupply2Status
+        tag: e_pdu2_device_status_power_supply2_status
+        mapping:
+          1: normal
+          2: alarm
+  - MIB: CPS-MIB
+    table:
+      OID: 1.3.6.1.4.1.3808.1.1.6.4.4
+      name: ePDU2PhaseStatusTable
+    symbols:
+      - OID: 1.3.6.1.4.1.3808.1.1.6.4.4.1.5
+        name: cyberpower.ePDU2PhaseStatusLoad
+      - OID: 1.3.6.1.4.1.3808.1.1.6.4.4.1.6
+        name: cyberpower.ePDU2PhaseStatusVoltage
+      - OID: 1.3.6.1.4.1.3808.1.1.6.4.4.1.7
+        name: cyberpower.ePDU2PhaseStatusPower
+      - OID: 1.3.6.1.4.1.3808.1.1.6.4.4.1.8
+        name: cyberpower.ePDU2PhaseStatusApparentPower
+      - OID: 1.3.6.1.4.1.3808.1.1.6.4.4.1.9
+        name: cyberpower.ePDU2PhaseStatusPowerFactor
+      - OID: 1.3.6.1.4.1.3808.1.1.6.4.4.1.10
+        name: cyberpower.ePDU2PhaseStatusPeakLoad
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.6.4.4.1.1
+          name: ePDU2PhaseStatusIndex
+        tag: e_pdu2_phase_status_index
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.6.4.4.1.4
+          name: ePDU2PhaseStatusLoadState
+        tag: e_pdu2_phase_status_load_state
+        mapping:
+          1: low_load
+          2: normal
+          3: near_overload
+          4: overload
+  - MIB: CPS-MIB
+    table:
+      OID: 1.3.6.1.4.1.3808.1.1.6.5.4
+      name: ePDU2BankStatusTable
+    symbols:
+      - OID: 1.3.6.1.4.1.3808.1.1.6.5.4.1.5
+        name: cyberpower.ePDU2BankStatusLoad
+      - OID: 1.3.6.1.4.1.3808.1.1.6.5.4.1.6
+        name: cyberpower.ePDU2BankStatusPeakLoad
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.6.5.4.1.1
+          name: ePDU2BankStatusIndex
+        tag: e_pdu2_bank_status_index
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.6.5.4.1.4
+          name: ePDU2BankStatusLoadState
+        tag: e_pdu2_bank_status_load_state
+        mapping:
+          1: low_load
+          2: normal
+          3: near_overload
+          4: overload
+  - MIB: CPS-MIB
+    table:
+      OID: 1.3.6.1.4.1.3808.1.1.6.6.1.4
+      name: ePDU2OutletSwitchedStatusTable
+    symbols:
+      - name: cyberpower.ePDU2OutletSwitchedStatus
+        constant_value_one: true
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.6.6.1.4.1.1
+          name: ePDU2OutletSwitchedStatusIndex
+        tag: e_pdu2_outlet_switched_status_index
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.6.6.1.4.1.4
+          name: ePDU2OutletSwitchedStatusName
+        tag: e_pdu2_outlet_switched_status_name
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.6.6.1.4.1.5
+          name: ePDU2OutletSwitchedStatusState
+        tag: e_pdu2_outlet_switched_status_state
+        mapping:
+          1: outlet_status_on
+          2: outlet_status_off
+  - MIB: CPS-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.3808.1.1.4.2.1.0
+      name: cyberpower.envir_temperature
```

Comment:
Out of curiosity, is there a reason we expose the fahrenheit but not the celsius? Does our UI handle unit switching?

Reply:
Good call out, our UI can handle the unit switching but only if the metadata csv is configured with the right unit or if the user configures it. Otherwise the value is shown without transformation.

It could make sense to submit only the celsius one, or both the celsius + the fahrenheit one.